sites/fast-website/src/app/index.ts

```diff
@@ -0,0 +1 @@
+export * from "@microsoft/fast-components";
```

Comment:
Does this get tree shaken if webpack has the production mode set?

Reply:
I tested a prod build (see a450e0b) and confirm that webpack doesn't drop the exports and the components on the page load and work as expected.

Reply:
Awesome, thanks!